spectrocloud/schemas/cluster_location_config.go

```diff
@@ -6,6 +6,21 @@ func ClusterLocationSchema() *schema.Schema {
 	return &schema.Schema{
 		Type:     schema.TypeList,
 		Optional: true,
+		DiffSuppressFunc: func(k, oldValue, newValue string, d *schema.ResourceData) bool {
```

Comment:
I suggest extracting this function separately so we can unit-test it in the future.

Reply:
And if possible implement unit tests now.

Reply:
I have created separate ticket to add unit test https://spectrocloud.atlassian.net/browse/PLT-312 
we can merge this PR